Python/errors.c

```diff
@@ -448,6 +447,15 @@ PyErr_GetExcInfo(PyObject **p_type, PyObject **p_value, PyObject **p_traceback)
     Py_XINCREF(*p_traceback);
 }
 
+
+void
+PyErr_GetExcInfo(PyObject **p_type, PyObject **p_value, PyObject **p_traceback)
+{
+    PyThreadState *tstate = _PyThreadState_GET();
+
```

Comment:
extreme nitpick: please remove this empytline.

Reply:
✅